utils/clientId.go

```diff
@@ -18,8 +18,11 @@ func GenerateClientIDFromRequest(req *http.Request) string {
 	clientID := strings.Join(ipAddressComponents, ":") + req.UserAgent()
 
 	// Create a MD5 hash of this ip + useragent
-	hasher := md5.New()
-	hasher.Write([]byte(clientID))
+	hasher := md5.New() //nolint
+	_, err := hasher.Write([]byte(clientID))
+	if err != nil {
+		log.Fatalln(err)
+	}
 	return hex.EncodeToString(hasher.Sum(nil))
```

Comment:
Not a bug, but you could trim this section down to:
```go
	clientID := strings.Join(ipAddressComponents, ":") + req.UserAgent()

	return hex.EncodeToString(md5.Sum([]byte(clientID))) //nolint
```

Reply:
This way looks a lot cleaner 👏

Reply:
I'm actually not able to get that work because `md5.Sum()` returns back a statically sized byte slice, `[16]byte`, and `hex.EncodeToString` doesn't like that.  I'm not sure why?  How is a certain sized slice still not a slice of that type?

Reply:
Just create a copy of it, like:

```go
b := md5.Sum([]byte(clientID))
hex.EncodeToString(b[:])
```

Reply:
Nice!  Thanks for the tip.